### DISCUSSION TITLE
install.sh: flag or check for env variable to skip root file system resizing

### DISCUSSION THREAD

**Discussion by herrfrei** [marcone/teslausb#906] [Ideas]
At least for my setup, resizing is not necessary or useful. I use an eMMC module for the operating system and an SD card for the data. I have created an environment variable analogous to `teslausb_setup_variables.conf` (`DATA_DRIVE`), whose existence is checked in install.sh and then the resizing is skipped.

**Comment by marcone** [comment]
I think this is already solved by https://github.com/marcone/teslausb/commit/bbf0c76739b5aa74ec2e06f78eff54b99554b4a6

**Comment by herrfrei** [comment]
Cool, many thanks!

marcone ***@***.***> schrieb am So., 22. Dez. 2024, 19:53:
